sdk/keyvault/keyvault-keys/test/utils/testAuthentication.ts

```diff
@@ -8,7 +8,10 @@ import { env, record, RecorderEnvironmentSetup } from "@azure/test-utils-recorde
 import { uniqueString } from "./recorderUtils";
 import TestClient from "./testClient";
 
-export async function authenticate(that: any): Promise<any> {
+// Adding this to the source would change the public API.
+type ApIVersions = "7.0" | "7.1";
```

Comment:
casing nit: `ApiVersions`

Reply:
doh I copied it from existing keyvault test code without realizing the typo.

Reply:
fixed